examples/Cytoscape_Example.ipynb

```diff
@@ -1,74 +1,74 @@
-{
```

Comment:
all this pain suggests your repo is doing line ending trash... highly recommend running this in git bash:

```bash
git config --global core.autocrlf true
```

and then re-checking out the repo and running the linter.

Reply:
Bumping this comment for @lukasnijhawan to address.